fable/src/fable/schema/variant.cpp

```diff
@@ -75,7 +74,7 @@ Variant::Variant(std::string desc, std::vector<Box>&& vec)
 }
 
 Json Variant::usage() const {
-  auto required = required_ ? "!" : "";
+  const char* required = required_ ? "!" : "";
   if (desc_.empty()) {
     return type_string() + required;
```

Comment:
Using `fmt::format` here is arguably more work for such a simple case. If I make this change here, then I should make the same change everwhere.

I'd prefer to limit my use of fmt to places where it really provides value, in case we ever want to provide a variant of the lib without the fmt dependency.

Reply:
fair enough. although fmt is (at least partially) getting absorbed into the stl :-)

Reply:
That will be a great day indeed!